tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmTypeReferenceUtil.xtend

```diff
@@ -0,0 +1,192 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.tcl.dsl.jvmmodel
+
+import com.google.gson.JsonArray
+import com.google.gson.JsonElement
+import com.google.gson.JsonObject
+import com.google.gson.JsonPrimitive
+import javax.inject.Inject
+import org.eclipse.emf.ecore.resource.Resource
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.xtext.common.types.JvmTypeReference
+import org.eclipse.xtext.common.types.JvmUnknownTypeReference
+import org.eclipse.xtext.xbase.jvmmodel.JvmTypeReferenceBuilder
+import org.eclipse.xtext.xbase.typesystem.conformance.TypeConformanceComputationArgument
+import org.eclipse.xtext.xbase.typesystem.references.StandardTypeReferenceOwner
+import org.eclipse.xtext.xbase.typesystem.util.CommonTypeComputationServices
+
+/**
+ * provide basic utility functions for JvmTypeReference(s)
+ * 
+ * since most functions make use of the type reference builder, please make sure
+ * that 'initWith' is called with the current resouce or resouceSet to allow classpath resolution
+ * of all relevant types.
+ */
+class TclJvmTypeReferenceUtil {
+
+	public val checkWithoutBoxing = new TypeConformanceComputationArgument(false, false, false, false, false, true)
```

Comment:
Why is this public? If access is required please annotate it with `@Accessors(PUBLIC_GETTER)`